src/cascadia/TerminalCore/lib/terminalcore-lib.vcxproj

```diff
@@ -8,6 +8,10 @@
     <ConfigurationType>StaticLibrary</ConfigurationType>
     <WindowsTargetPlatformMinVersion>10.0.17763.0</WindowsTargetPlatformMinVersion>
     <RootNamespace>Microsoft.Terminal.TerminalControl</RootNamespace>
+
+    <!-- sets a bunch of Windows Universal properties -->

```

Comment:
would that we explained why we needed this! i hate that it's a magic giant hammer that fixes weird problems.